queenbee/recipe/recipe.py

```diff
@@ -65,6 +72,30 @@ class Recipe(BaseModel):
 
     @classmethod
     def from_folder(cls, folder_path: str):
+        """Generate a recipe from a folder
+
+        Note:
+            Here is an example of a Recipe folder:
+            ::
+
+                .
+                ├── .dependencies
+                │   ├── operators
+                │   │   └── <sha-256>.yaml
+                │   └── recipes
+                │       └── <sha-256>.yaml
```

Comment:
I assume this will be generated by a command? Do we have some documentation for this. I like the idea of how you are keep tracking on the unique items. 👍

Reply:
All part of the joys of building a package manager :upside_down_face: